src/types/utils.cpp

```diff
@@ -521,6 +521,44 @@ catch (...)
     return {};
 }
 
+// Routine Description:

```

Comment:
This may not be the correct place to put this function, but I wanted somewhere that both `SettingsEditor` and `QueryExtension` could access.